Lib/fontTools/misc/intTools.py

```diff
@@ -1,14 +1,25 @@
-__all__ = ['popCount']
+__all__ = ["popCount"]
```

Comment:
Just wondering. Any reason you prefer double-quotes over single? :) Since they are same in Python, one can argue that the single wins since doesn't require shifting in QWERTY. :)

Reply:
It's the default in black :O and since me and Cosimo want to blacken the source _at some point_....

Reply:
I see.

Reply:
Do it after this lands. I don't think there's any big outstanding PRs.

Reply:
I need to fiddle usort and black into place and then the blackening and sorting can begin... maybe when I get the time.

Reply:
Is usort the new black? (funny my fingers auto type "ushort" instead :)))